incubator/drone/templates/deployment-server.yaml

```diff
@@ -36,9 +36,9 @@ spec:
                 key: secret
           - name: DRONE_HOST
           {{- if hasKey .Values.server "host" }}
-            value: {{ .Values.server.host }}
+            value: "http://{{ .Values.server.host }}"
```

Comment:
I think that this should be removed: it is up to the user to decide which scheme to use. This could result in an unexpected behavior.

Reply:
It's true, but then it could be IMO a also a value parameter e.g. `.Values.server.scheme` which defaults to `http`.

Reply:
I think that adding a comment before server.host and in the README should be enough.

Reply:
Agree. Or maybe in `NOTES.txt` which is quite nice.

Reply:
Yeah, I'll remove this in a bit. I did not see that the the [suggested value in `values.yaml`](https://github.com/kubernetes/charts/blob/3eb083669c95ce619520c67dcc87564dbdb1267d/incubator/drone/values.yaml#L65-L67) already included a scheme. I'll also add comments there and in the `README`